flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/windowing/WindowOperatorTest.java

```diff
@@ -2550,4 +2550,36 @@ public String toString() {
 			return "EventTimeTrigger()";
 		}
 	}
+
+	@Test
```

Comment:
Very good that you added a test for this! 👍 

I'd like to have these in a separate test for `TimeWindow`, however, since the `WindowOperatorTest` file is already very big.